android/src/main/kotlin/com/kasem/receive_sharing_intent/ReceiveSharingIntentPlugin.kt

```diff
@@ -143,7 +143,7 @@ class ReceiveSharingIntentPlugin : FlutterPlugin, ActivityAware, MethodCallHandl
         return when (intent.action) {
             Intent.ACTION_SEND -> {
                 val uri = intent.getParcelableExtra<Uri>(Intent.EXTRA_STREAM)
-                val path = FileDirectory.getAbsolutePath(applicationContext, uri)
+                val path = FileDirectory.getAbsolutePath(applicationContext, uri!!)
```

Comment:
Use null safe instead. i.e. ` val path = uri?.let{ FileDirectory.getAbsolutePath(applicationContext, it) }`

Reply:
Fixed